FirstTask.py

```diff
@@ -0,0 +1,180 @@
+from os import system as sys
+
+class BankAccount:
+    def __init__(self, Name, PassCode, AccNumber, Balance, AccoutType):
+        self.Name = Name
+        self.Passcode = PassCode
+        self.Balance = Balance
+        self.AccountType = AccoutType
+        self.AccNumber = AccNumber
+
+    def FullName(self):
+        return self.Name
+
+    def AccountNumber(self):
+        return self.AccNumber
+
+    def PassCode(self):
+        return self.Passcode
+
+    def CheckBalance(self):
+        return self.Balance
+
+    def CheckAccountType(self):
+        return self.AccountType
+
+    def AccDetails(self):

```

Comment:
Every method name should be in snake-case notation. This can be `acc_details` and all other method names below should also be updated.